src/tox/Network/Tox.lhs

```diff
@@ -158,7 +158,7 @@ peer from which it receives one of these packets.  Through this mechanism, Tox
 clients will bootstrap themselves automatically from other Tox clients running
 on the local network.
 
-Toxcore sends these packets every 10 seconds to keep delays low.  The packets
+When enabled toxcore sends these packets every 10 seconds to keep delays low.  The packets
```

Comment:
`When enabled, toxcore` (missed comma, because I know @iphydf is gonna ask for it.)